src/main/java/com/puppycrawl/tools/checkstyle/AuditEventDefaultFormatter.java

```diff
@@ -100,24 +100,10 @@ private static int calculateBufferLength(AuditEvent event, int severityLevelName
      */
     private static String getCheckShortName(AuditEvent event) {
         final String checkFullName = event.getSourceName();
-        final String checkShortName;
-        final int lastDotIndex = checkFullName.lastIndexOf('.');
-        if (lastDotIndex == -1) {
-            if (checkFullName.endsWith(SUFFIX)) {
-                checkShortName = checkFullName.substring(0, checkFullName.lastIndexOf(SUFFIX));
-            }
-            else {
-                checkShortName = checkFullName;
-            }
-        }
-        else {
-            if (checkFullName.endsWith(SUFFIX)) {
-                checkShortName = checkFullName.substring(lastDotIndex + 1,
-                    checkFullName.lastIndexOf(SUFFIX));
-            }
-            else {
-                checkShortName = checkFullName.substring(lastDotIndex + 1);
-            }
+        String checkShortName = checkFullName.substring(checkFullName.lastIndexOf('.') + 1);
```

Comment:
The method `String.substring` is smart enough to return `this` when `startIdx == 0`.